admin_guide/managing_pods.adoc

```diff
@@ -0,0 +1,259 @@
+
+As an {product-title} cluster administrator, you can use egress policy to limit
+the addresses that some or all pods can access from within the cluster, so that:
+
+- A pod can only talk to internal hosts, and cannot initiate connections to the
+public Internet.
++
+Or,
+- A pod can only talk to the public Internet, and cannot initiate connections to
+internal hosts (outside the cluster).
++
+Or,
+- A pod cannot reach specified internal subnets/hosts that it should have no
+reason to contact.
+
+For example, you can configure projects with different egress policies, allowing
+`<project A>` access to a specified IP range, but denying the same access to
+`<project B>`.
+
+[CAUTION]
+====
+You must have the
+xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[*ovs-multitenant* plug-in] enabled in order to limit pod access via egress policy.
+====
+
+Project administrators can neither create `*EgressNetworkPolicy*` objects, nor
+edit the ones you create in their project.
+
+The `default` project (and any other global namespace) cannot have egress
+policy.
+
+[NOTE]
+====
+If you merge two projects together (via `oadm pod-network join-projects`),
+then you cannot use egress policy in _any_ of the joined projects.
+
+If you make a project global (via `oadm pod-network make-projects-global`),
+then it cannot have an `*EgressNetworkPolicy*`.
+
+If an allowed network overlaps with a denied network, then the rules are
+checked in order, and the first one that matches is enforced.
+====
+
+[[admin-guide-config-pod-access]]
+=== Configuring Pod Access Limits
+
+To configure pod access limits, you must use the `oc` command or the REST API.
+You can use `oc [create|replace|delete]` to manipulate `*EgressNetworkPolicy*`
+objects. The *_api/swagger-spec/oapi-v1.json_* file has API-level details on how
+the objects actually work.
+
+To configure pod access limits:
+
+. Navigate to the project you want to affect.
+. Create a JSON file for the pod limit policy:
++
+----
+# oc create -f <policy>.json
+----
+. Configure the JSON file with policy details. For example:
++
+----
+{
+    "kind": "EgressNetworkPolicy",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "default"
+    },
+    "spec": {
+        "egress": [
+            {
+                "type": "Allow",
+                "to": {
+                    "cidrSelector": "1.2.3.0/24"
+                }
+            },
+            {
+                "type": "Deny",
+                "to": {
+                    "cidrSelector": "0.0.0.0/32"
+                }
+            }
+        ]
+    }
+}
+----
++
+When the example above is added in a project, it allows traffic to `1.2.3.0/24`,
+but denies access to all other external IP addresses. This would not affect
+traffic to other pods.
+
+[[admin-guide-manage-pods-limit-bandwidth]]
+== Limiting the Bandwidth Available to Pods
+
+You can apply quality-of-service traffic shaping to a pod and effectively limit
```

Comment:
I have no idea what "quality-of-service traffic shaping" is. Is it really just "limiting the available bandwidth"? If so, could we get rid of the first part and just get to the second? The argument against this is if it's an established term and I'm just not getting it cos I'm not hip to the lingo.